libnmstate/route_rule.py

```diff
@@ -38,11 +38,11 @@ def _complement_defaults(self):
         ):
             self.route_table = KERNEL_MAIN_ROUTE_TABLE_ID
 
-    def _append_prefix_length_for_host_only_ip(self):
-        if self.ip_from and "/" not in self.ip_from:
-            self.ip_from = to_ip_address_full(self.ip_from)
-        if self.ip_to and "/" not in self.ip_to:
-            self.ip_to = to_ip_address_full(self.ip_to)
+    def _canonicalize_ip_address(self):
+        if self.ip_from:
+            self.ip_from = canonicalize_ip_network(self.ip_from)
+        if self.ip_to:
+            self.ip_to = canonicalize_ip_network(self.ip_to)
```

Comment:
IMO, this should be `_canonicalize_ip_network(self)` as the method is only canonicalizing ip networks not a single address.

Reply:
I mean, to rename the function to that.